src/material-experimental/mdc-chips/chips.scss

```diff
@@ -100,6 +100,19 @@ input.mat-mdc-chip-input {
   flex: 1 0 $mat-chip-input-width;
 }
 
+// The margin value is set in MDC.
+$mat-mdc-chip-margin: 4px;
```

Comment:
MDC has some variables for the margin. Could we derive this value from them?

Reply:
That would be cool! I don't have a ton of css experience, but it looks to me like it would be possible, but only with some refactoring in MDC. I think the 4px value is derived from the gap-size parameter here: https://github.com/material-components/material-components-web/blob/master/packages/mdc-chips/_mixins.scss#L510, and since we use the core styles, our gap-size value is just hardcoded to 8px here: https://github.com/material-components/material-components-web/blob/master/packages/mdc-chips/_mixins.scss#L319 I don't see a way for us to reference that value as it is. Maybe something nice to do for the future, but out of scope for this change?

Reply:
I was referring to the ones [here](https://github.com/material-components/material-components-web/blob/master/packages/mdc-chips/_variables.scss#L510), but it seems like they're primarily for icons. Let's keep it as it is for now.